<ui version="4.0" >
 <class>LightingManagerClass</class>
 <widget class="QWidget" name="LightingManagerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>LightingManager</string>
  </property>
  <widget class="QGroupBox" name="gbMainFrame" >
   <property name="geometry" >
    <rect>
     <x>-8</x>
     <y>0</y>
     <width>541</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >QGroupBox{
	background-color: qlineargradient( x1: 0, y1: 0, x2:0.5, x2: 0.5, stop: 0 #FF92BB, stop: 1 white);
}
</string>
   </property>
   <widget class="QPushButton" name="pbResetTotalSumHour" >
    <property name="geometry" >
     <rect>
      <x>400</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS UI Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet" >
     <string notr="true" >font-family: MS UI Gothic;font-size: 11pt;font-weight: bold;</string>
    </property>
    <property name="text" >
     <string>累計時間リセット</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTotalSumHour" >
    <property name="geometry" >
     <rect>
      <x>188</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="numDigits" >
     <number>6</number>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="pgbElapsedTime" >
    <property name="geometry" >
     <rect>
      <x>18</x>
      <y>90</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>8</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS UI Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet" >
     <string notr="true" >font-family: MS UI Gothic;font-size: 20pt;font-weight: bold;</string>
    </property>
    <property name="text" >
     <string>累計時間</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbMainMessage" >
    <property name="geometry" >
     <rect>
      <x>8</x>
      <y>20</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS UI Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet" >
     <string notr="true" >font-family: MS UI Gothic;font-size: 20pt;font-weight: bold;</string>
    </property>
    <property name="text" >
     <string>照明が安定するまでしばらくお待ち下さい．．．</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdElapsedTime" >
    <property name="geometry" >
     <rect>
      <x>188</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="numDigits" >
     <number>4</number>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDetail" >
    <property name="geometry" >
     <rect>
      <x>448</x>
      <y>220</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>詳細</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS UI Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet" >
     <string notr="true" >font-family: MS UI Gothic;font-size: 20pt;font-weight: bold;</string>
    </property>
    <property name="text" >
     <string>分で完了</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>328</x>
      <y>200</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS UI Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet" >
     <string notr="true" >font-family: MS UI Gothic;font-size: 20pt;font-weight: bold;</string>
    </property>
    <property name="text" >
     <string>時間</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>8</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS UI Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet" >
     <string notr="true" >font-family: MS UI Gothic;font-size: 20pt;font-weight: bold;</string>
    </property>
    <property name="text" >
     <string>あと</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>280</y>
     <width>511</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pbShowMessage" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text" >
     <string>メッセージ表示テスト</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIsShowWidget" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>20</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>起動時に画面を表示する</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>暖機時間</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>照明交換目安時間</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbWarmMinute" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbChangeHour" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>時間</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIsCloseWidget" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>50</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>暖機終了後、画面を閉じる</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbIsStartExe" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>491</width>
      <height>54</height>
     </rect>
    </property>
    <property name="title" >
     <string>暖気終了後、起動するソフトを指定する</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLineEdit" name="leStartExe" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QToolButton" name="tbStartExe" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
